README.md

```diff
@@ -475,7 +475,7 @@ Useful behind (corporate) proxies that act like man-in-the middle on https conne
 
 - "**token**" - Github OAuth token:
 
-The OAuth token can be used to boost the Github API rate-limit from 60 to 5000 (non-cached) requests per hour. The is token needs just ['read-only access to public information'](http://developer.github.com/v3/oauth/#scopes) so no additional OAuth scopes are necessary.
+The OAuth token can be used to boost the Github API rate-limit from 60 to 5000 (non-cached) requests per hour. The token needs just ['read-only access to public information'](http://developer.github.com/v3/oauth/#scopes) so no additional OAuth scopes are necessary.
```

Comment:
The phrase "The token needs just" doesn't quite make sense still. Did you want to change it to read "The token only needs 'read-only access to public information', no additional OAuth scopes are necessary."?